Documentation/network/concepts/ipam/azure.rst

```diff
@@ -185,6 +185,33 @@ interface:
 This means that the number of IPs allocated in a single allocation cycle can be
 less than what is required to fulfill ``spec.ipam.pre-allocate``.
 
+Static Public IP Allocation
+----------------------------
+
+Nodes can be assigned static public IPs from tagged Azure Public IP Prefixes.
+
+1. Create and tag a `Public IP Prefix <https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-address-prefix>`__
+   in the same Resource Group as your nodes::
+
+      az network public-ip prefix create \
+        --resource-group $RESOURCE_GROUP \
+        --name $PREFIX_NAME \
+        --length 28 \
+        --tags prefix-tag-key=prefix-tag-value
```

Comment:
This should give you proper highlighting for the snippet (admittedly, this won't change much for this one)

```suggestion
   .. code-block:: shell-session
   
      $ az network public-ip prefix create \
        --resource-group $RESOURCE_GROUP \
        --name $PREFIX_NAME \
        --length 28 \
        --tags prefix-tag-key=prefix-tag-value
```